### ISSUE TITLE
Can't reorder or unpin feeds

### ISSUE THREAD

**Issue by j4ckxyz** [bluesky-social/social-app#5927]
### Steps to Reproduce

1. Go to "Feeds"
2. Edit
3. Reorder or unpin any feed
4. Click "Save changes"
5. Should receive a popup on lower left which says "There was an issue contacting the server"

### Attachments

<img width="1238" alt="image" src="https://github.com/user-attachments/assets/b400d68b-d644-4c2d-a37b-ed1d2dca1229">
<img width="1342" alt="image" src="https://github.com/user-attachments/assets/7ab77c98-5644-4e4f-b483-c2e1f92dc293">
<img width="1785" alt="image" src="https://github.com/user-attachments/assets/1f37bdb4-4582-4ab8-8f76-85e1dbb2ac03">
First screenshot is my feeds before, second is once I have reordered them and unpinned some (happens on any feed, not a feed-specific issue..), third screenshot is the message I get when I click save.

Here it was done on desktop on Microsoft Edge, but I have this issue across all browsers I've tried (Chromium AND FF, as well as Safari) & happens on mobile, too.

### What platform(s) does this occur on?

Web (Desktop)

### Device Info

Mac mini - M2 8/256, macOS Sequoia 15.2 Developer Beta, Microsoft Edge.

### What version of the app are you using?

Build version: 1.92.1; Bundle info: 89cff31 (prod); Bundle date: 24101819; Platform: web

### Additional Information

https://github.com/user-attachments/assets/2336ef87-b05e-467d-88ed-d424d06b3b64

Here's the console as it happens, it has some blocked scripts but that's just uBlock stopping the data collection service which Bluesky uses. Disabling uBlock did **not** fix the issue :(

**Comment by skyline385** [comment]
Can confirm that this bug is still there, using the same steps as above. I haven't been able to organize my feed in months because of it.

**Comment by AndEliRich** [comment]
I have this issue as well. It occurs on the iPhone app and web. I have not been able to reorganize feeds for quite some time. Hope it is fixed soon.

**Comment by alexswabey** [comment]
Still getting it on the web and mobile client.

**Comment by AndEliRich** [comment]
Yes, this bug still exists for me too on all platforms. So annoying!

**Comment by jonesyriffic** [comment]
Seeing this bug on all platforms. Appears to be account-related rather than app related though as it affects my main account only.

**Comment by thebruge** [comment]
I have this same issue on both iOS and Web client, anything you need from me to help debug?

**Comment by devaunbeats** [comment]
I have also had this exact issue for months and there's been no word about a fix. If I want to reorganize my feeds I literally have to remove all of them one by one and then add them one by one. 

They really need to fix this

**Comment by thebruge** [comment]
> I have also had this exact issue for months and there's been no word about a fix. If I want to reorganize my feeds I literally have to remove all of them one by one and then add them one by one.
> 
> They really need to fix this

I got access to the new Flashes app, this allows me to reorder my feeds and they are reflected in the main BlueSky app. Not ideal workaround but less intrusive. We still need a fix for this though!

**Comment by CuriousDenise** [comment]
Same thing is happening to me on iOS and web versions of Bluesky.

**Comment by CuriousDenise** [comment]
> > I have also had this exact issue for months and there's been no word about a fix. If I want to reorganize my feeds I literally have to remove all of them one by one and then add them one by one.
> > They really need to fix this
> 
> I got access to the new Flashes app, this allows me to reorder my feeds and they are reflected in the main BlueSky app. Not ideal workaround but less intrusive. We still need a fix for this though!

I just did the same thing (reorder in Flashes) and it reordered in Flashes, but not in Bluesky. (BTW, leaving Flashes and going back in showed me updated # Feeds in Saved, but my Feeds was completely blank).

**Comment by j4ckxyz** [comment]
Oh, btw I managed to fix it myself by opening each feed manually and unpinning it from the menu at the top, then click "Unpin Feed". It's annoying, and definitely not the real fix, but it seemed to remove the few feeds that caused bugs (whichever they were.)

<img width="458" alt="Image" src="https://github.com/user-attachments/assets/4689f24c-a95c-4993-8a79-80aff34a36ea" />

**Comment by Ev-Abbott** [comment]
Thanks for folks who posted workarounds! This has been painful on my end. Would absolutely love a fix here if possible!

**Comment by samlfair** [comment]
+1 on this. I've had the same issue for weeks.

**Comment by o-simardcasanova-net** [comment]
Hi all. If you still experience this issue, I've found a [partial workaround](https://github.com/bluesky-social/social-app/issues/7436#issuecomment-2870051495).

**Comment by mozzius** [comment]
I believe this is fixed by v1.104

Basically, the utility functions for saving feeds validate that the feed IDs are TIDs, but 3rd party apps have a habit of using non-TID identifiers. We removed this useless check, so it should work now
